public/locales/en/notify.json

```diff
@@ -5,5 +5,7 @@
   "windowIpfsRequestFailed": "IPFS request failed. Please check your IPFS Companion settings.",
   "ipfsIsBack": "Normal IPFS service has resumed. Enjoy!",
   "filesEventFailed": "Failed to add files to IPFS. Please try again.",
-  "folderExists": "A folder with that name already exists. Please choose another."
+  "folderExists": "A folder with that name already exists. Please choose another.",
+  "couldntConnectToSwarm": "Could not connect to the swarm.",
```

Comment:
Probably need to change this language a bit:
User is probably connected to the swarm already and is just trying to connect to one of members of the swarm, not THE swarm (that happens during bootstrap).

Reply:
Done!